SpatialGDK/Source/SpatialGDK/Private/Utils/ActorGroupManager.cpp

```diff
@@ -0,0 +1,64 @@
+#include "Utils/ActorGroupManager.h"
+#include "SpatialGDKSettings.h"
+
+void UActorGroupManager::Init()
```

Comment:
Maybe have more descriptive name than `Init`?
```suggestion
void UActorGroupManager::LoadWorkerToActorMapping()
```